packages/fake-api/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "nodenext",
+    "strict": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "rootDir": "./src"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules"]
+}
```

Comment:
When we were talking at some point we need to start combining tsconfig files, try and do something that is simiar to how the create-vue-app generator does it.